backend/dal/Migrations/20201223235542_01.06.00.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using Pims.Dal.Helpers.Migrations;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class _010600 : SeedMigration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<string>(
+                name: "EncumbranceReason",
+                table: "Parcels",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "EffectiveDate",
+                table: "ParcelFiscals",
+                type: "DATE",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "EncumbranceReason",
```

Comment:
You should edit the `BuildingConfiguration` to set the correct size of this field.